src/Presentation/RestSaveMappingsPresenter.php

```diff
@@ -46,6 +45,13 @@ public function presentInvalidEntityId(): void {
 		);
 	}
 
+	public function presentPermissionDenied( Throwable $exception ): void {
+		$this->response = $this->responseFactory->createHttpError(
+			403,
+			[ 'message' => $exception->getMessage() ]
```

Comment:
This is a bit different than the response you get with `responseFactory->createLocalizedHttpError()`. I will get back to this later once I figure out how to get a user displayable message in a consistent way.